### ISSUE TITLE
Write a step-by-step guide for using a board in an app

### ISSUE THREAD

**Issue by dglazkov** [breadboard-ai/breadboard#749]
The current [happy path]() has just enough info to create a board, but there's nothing there about how to fit into an app (like a web app). Let's write a guide for building a simple app from an existing board. Probably just plain HTML/CSS/JS or maybe with Lit/React.

**Comment by PaulKinlan** [comment]
We had a heap of demos from @Mearman and co - we could probably re-provsion some of them.

Btw - I think we will need two entry points:

1. I have a board, now scaffold an app
2. I have an app, how do I integrated breadboard

**Comment by Mearman** [comment]
For the most part, we got them working with the latest release. Though with the significant progress in the UI components and editor, it's building them from a clean slate.

@alexandraM98 has been getting to grips with the UI components, and @TinaNikou has been rewriting our getting-started examples using the new grammar.

I've requisitioned some of @jamesjacko and @jig925's time also next week. So, together with the rest of the @breadboard-ai/exadev team, we should be able to put our heads together and think about a path forward.

The way you've handled the proxy in the critic is an interesting scenario that I'm just going to link here as a bookmark. https://replit.com/@paulkinlan/VioletAfraidQuotes#ui-src/boards/the-panel.ts

This also aligns a bit with https://github.com/breadboard-ai/breadboard/issues/722

**Comment by jamesjacko** [comment]
I think the kit section of the happy path needs a little more work as it isn't immediately obvious where the kit fits into the existing code from previous steps. The example `ptomptTemplate` kit is also a little confusing on first use as the example code does nothing but return the string in the prompt. Maybe changing that example to something like:

``` typescript
const prompt = templates.promptTemplate({
    template: "The input reversed is {{text}}",
    text,
  });
```
I realize that the Happy Path is more about getting up and running with breadboard but using the `promptTemplate` and it only returning "Hello, !" is confusing for someone who is getting started, I think.

**Comment by dglazkov** [comment]
> I think the kit section of the happy path needs a little more work as it isn't immediately obvious where the kit fits into the existing code from previous steps.

I agree! Ideally, I'd love to rewrite the happy path guide entirely to follow the beats outlined here:
https://github.com/breadboard-ai/breadboard/issues/509
